DOCS-4011: Update manage-viam-server (#4334)
viamrobotics/docs: docs/manage/reference/viam-agent/manage-viam-agent.md

  This command uninstalls `viam-agent`, `viam-server`, the machine cloud credentials file (<file>/etc/viam.json</file>), and the provisioning configuration file (<file>/etc/viam-provisioning.json</file>).

>   {{< alert title="Caution" color="caution" >}}
>   If you remove the machine cloud credentials file you will not be able to connect to your machine without restoring it.
>   You can only restore this file if you have access to the machine in the Viam app.
>   {{< /alert >}}

Reviewer: ```suggestion
{{< alert title="Caution" color="caution" >}}
If you remove the machine cloud credentials file you will not be able to connect to your machine.
You can only restore this file if you have access to the machine configuration in the Viam app.
{{< /alert >}}
```
[s] I'm not sure if it has any impact on an admonition, but I don't think this should be indented.
[opt] 'without restoring it' -- took me a second to figure out what 'it' was referring to, the machine or the file. I think the second sentence does a good job of explaining this anyway and removing this clause arguably makes this sentence clearer!
